src/components/Comments/NewCommentForm.tsx

```diff
@@ -0,0 +1,129 @@
+import {
+  FC, FormEvent, useState, useEffect,
+} from 'react';
+import classNames from 'classnames';
+import { TextField } from './TextField';
+import { CommentData, Field, FieldIcon } from '../../types';
+import { useCommentsContext } from '../../hooks/useCommentsContext';
+import { useGlobalContext } from '../../hooks/useGlobalContext';
+
+export const NewCommentForm: FC = () => {
+  const { resetError } = useGlobalContext();
+  const { onAddNewComment, commentIsProcessing } = useCommentsContext();
+
+  const [userName, setUserName] = useState('');
+  const [userEmail, setUserEmail] = useState('');
+  const [userCommentText, setUserCommentText] = useState('');
+
+  const [isSubmitted, setIsSubmitted] = useState(false);
+  const [isReseted, setIsReseted] = useState(false);
+
+  const onSubmit = async (event: FormEvent) => {
+    event.preventDefault();
+    setIsSubmitted(true);
+
+    if (!userName || !userEmail || !userCommentText) {
```

Comment:
`trim` will fix this issue
<img width="587" alt="Screenshot 2023-08-13 at 6 37 06 PM" src="https://github.com/mate-academy/react_dynamic-list-of-posts/assets/60295799/2b21cb25-e76f-4adb-a037-532738e556ac">